crates/wasmtime/src/store.rs

```diff
@@ -1759,7 +1835,7 @@ unsafe impl<T> wasmtime_runtime::Store for StoreInner<T> {
         // self.async_cx() panicks when used with a non-async store, so
         // wrap this in an option.
         #[cfg(feature = "async")]
-        let async_cx = if self.async_support() {
+        let opt_async_cx = if self.async_support() {
             Some(self.async_cx())
```

Comment:
Ditto to other places, it's ok to `.unwrap()` here

Reply:
Addressed in a775e97.